### PR TITLE
Add Scenario docs for Connect to WSO2 API Manager as event hub to validate JWT subscriptions

### PR DESCRIPTION
**Pull request by Jayanie**
- Adds Scenario docs for Connect to WSO2 API Manager as event hub to validate JWT subscriptions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -525,7 +525,7 @@ and configurations related to API operator and API portal.
 1. [Sample 21: Deploy a service as a managed API using WSO2 API Manager locally](scenarios/scenario-21)
 1. [Sample 22: Generating Backend JWT](scenarios/scenario-22)
 1. [Sample 23: Add Configmaps and Secrets to Micro-gateway Deployment](scenarios/scenario-23)
-
+1. [Sample 24: Connect to WSO2 API Manager as event hub to validate JWT subscriptions](scenarios/scenario-24)
```
Comment (pubudu538) [inline README.md:528]: Lets use the topic using event hub in WSO2 API Manager.

File: scenarios/scenario-24/README.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+  
+ ##### Deploying the artifacts
+ 
+- Enabling for connecting to WSO2 API Manager as event hub for retrieving API and subscription data from API Manager
+can be configured in the apim-config in `controller-configs/controller_conf.yaml`.
+- You can set the value to true. 
+     ```
+     # Enable configurations for retrieving API and subscription data from API Manager.
+     enabledEventhub: "true"
+     ```
```
Comment (pubudu538) [inline scenarios/scenario-24/README.md:18]: Also, let's mention other configuration details such as connection url, etc.

File: scenarios/scenario-24/README.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
+## Scenario 24 - Connect to WSO2 API Manager as event hub to validate JWT subscriptions
```
Comment (pubudu538) [inline scenarios/scenario-24/README.md:1]: We have to mention that we need to run API manager in K8s or anywhere that has access from MG. Also mention that in this tutorial that we are using the API manager on K8s and show steps how to deploy APIM on K8s. We can point here - https://github.com/wso2/K8s-api-operator#step-4-install-the-api-portal-and-security-token-service

**Review by pubudu538 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Jayanie) [reply README.md]: Fixed by 2870f18

File: scenarios/scenario-24/README.md (hunk 3 above)
Comment (Jayanie) [reply scenarios/scenario-24/README.md]: Fixed by 4b0a784

File: scenarios/scenario-24/README.md (hunk 2 above)
Comment (Jayanie) [reply scenarios/scenario-24/README.md:40]: Fixed by b573db6

**Review by pubudu538 — APPROVED** [review]

**Pull request merged**